src/Collections-Abstract-Tests/TPrintTest.trait.st

```diff
@@ -37,13 +37,12 @@ TPrintTest >> testPrintElementsOn [
 { #category : #'tests - printing' }
 TPrintTest >> testPrintNameOn [
 
-	| aStream result |
-	result:=''.
-	aStream:= ReadWriteStream on: result.
-	self nonEmpty printNameOn: aStream.
+	| printed |
+	printed := String streamContents: [ :stream | self nonEmpty printNameOn: stream ].
+
 	self nonEmpty class name first isVowel
-		ifTrue:[ self assert: aStream contents =('an ',self nonEmpty class name ) ]
-		ifFalse:[self assert: aStream contents =('a ',self nonEmpty class name)]
+		ifTrue: [ self assert: printed equals: 'an ' , self nonEmpty class name ]
+		ifFalse: [ self assert: printed equals: 'a ' , self nonEmpty class name ]
```

Comment:
This could also be:

```st
self assert: printed equals: (self nonEmpty class name first isVowel ifTrue: [ 'an ' ] ifFalse: [ 'a ']) , self nonEmpty class name
```

Reply:
yes, i didn't want to change everything

Reply:
done.